scipy/stats/_continuous_distns.py

```diff
@@ -964,7 +964,10 @@ def _munp(self, n, a, b):
                               ((b - 4.0)*(b - 3.0)*(b - 2.0)*(b - 1.0))),
                 fillvalue=np.inf)
         else:
-            raise NotImplementedError
+            return _lazywhere(b > n, (a, b),
+                              # sup=super() prevents late-binding issue
+                              lambda a, b, sup=super(): sup._munp(n, a, b),
+                              fillvalue=np.inf)
```

Comment:
We should just use the general formula here.

![image](https://github.com/scipy/scipy/assets/6570539/f7002338-2939-4082-ae6b-ceaade090942)